--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -29,9 +29,9 @@
 TARGET_LINK_LIBRARIES(json_verify yajl_s)
 
 # copy in the binary
-GET_TARGET_PROPERTY(binPath json_verify LOCATION)
+GET_TARGET_PROPERTY(binPath json_verify yajl-2.1.1)
 
-ADD_CUSTOM_COMMAND(TARGET json_verify POST_BUILD
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${binPath} ${binDir})
+#ADD_CUSTOM_COMMAND(TARGET json_verify POST_BUILD
+#    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${binPath} ${binDir})
 
 INSTALL(TARGETS json_verify RUNTIME DESTINATION bin)
